### DISCUSSION TITLE
Channel on IRC on Libera.chat server

### DISCUSSION THREAD

**Discussion by vlcg-gif** [carbon-language/carbon-lang#3031] [General]
A channel was created for conversations in real time with other users in IRC, just enter:
https://web.libera.chat/#carbon-lang
Choose a nickname and enjoy!

**Comment by geoffromer** [comment]
Hey, in case you missed it, Carbon has an [official Discord server](https://discord.gg/ZjVdShJDAs) for real-time chat. If you have any concerns with Discord that we might be able to address, please let us know.

**Comment by vlcg-gif** [comment]
For privacy and security reasons I prefer IRC

**Reply by chandlerc** [reply]
Understood, but we're careful to only have official discussion spaces where the Carbon community is well set up to moderate and make sure that space remains welcoming and friendly.

Unfortunately, we don't realistically have the bandwidth on the team to moderate a second real-time chat space right now, and need to focus our efforts moderating a single, specific one. The moderation tools on Discord are also *substantially* more effective in our experience, and so that's where we're currently focusing our efforts.

Folks are more then welcome to discuss Carbon wherever they feel most comfortable of course, but I think we can't really expand the official chat spaces right now to include IRC.

That said, real-time chat is super *useful* but it isn't *necessary* to contribute. You can always use GitHub to discuss and contribute to Carbon if that works better for you.

**Reply by vlcg-gif** [reply]
Okay, but if you need to, you can count on me to help, I'm at your disposal. I really like IRC because its instant messaging service is decentralized and more independent. But that's not the point now, if you need help taking it to IRC let me know too, thanks man!

**Reply by chandlerc** [reply]
If we get to the point of expanding, will do, and thanks for the offer!
